hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/MiscCryptoUtils.java

```diff
@@ -49,6 +51,12 @@ public static byte[] keccak256DigestOf(final byte[] msg) {
         return new Keccak.Digest256().digest(msg);
     }
 
+    public static Bytes keccak256DigestOf(final Bytes msg) {
+        final MessageDigest digest = new Keccak.Digest256();
```

Comment:
Is this method used from multiple threads? If not, this digest object can be changed to static, and its reset() method is called before every call to digest()

Reply:
Yes, it is called from multiple threads for transaction signature verification. The constructor's cost is rather low.